aspnet/fundamentals/owin/sample/src/OwinSample/Startup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNet.Builder;
```

Comment:
Were these samples copied unmodified from Entropy? In some places your docs reference these, but in others they reference Entropy directly.

@danroth27  How do we plan to maintain these separate copies?